.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,27 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-version: [ '7.4' ]
```

Comment:
I still suggest keeping `8.0` and `8.1` here.
They will start working with no additional changes as soon as https://github.com/oat-sa/generis/pull/1017 and https://github.com/oat-sa/tao-core/pull/3573 have been merged.

Otherwise, we will have to update the workflow again.

Reply:
Why would we have to update workflow? Why can't we add this when those branches are merged?